### DISCUSSION TITLE
Updating to pre-release versions of .NET (possibly others)

### DISCUSSION THREAD

**Discussion by adusak** [renovatebot/renovate#10807] [Request Help]
Hello, I have a quick question about upgrade versions.
So couple of months ago renovate started suggesting we upgrade our base docker images to pre-release versions of .NET
Back then I had to revert to using version 24.119.16 which does not have this behavior.

I retested today with the latest version, and it seems that the issue has not disappeared and is probably the intended behavior.

Here is the screenshot of the PR created today:
![image](https://user-images.githubusercontent.com/1461364/125432799-f1141006-55bb-44af-a49b-1a99c668832a.png)

Is there a way to disable this behavior?

**Comment by rarkins** [comment]
Renovate previously did not propose Docker major updates by default. Now it does, and it's intentional. 

It's just coincidental that this image has a unique versioning approach where you can't infer stability from the version itself. 

You can close the PR to ignore it and should not get any future v5-v6 upgrades proposed.
